src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,374 @@
+    }
+
+    private static String[] splitEventDescriptionAndDate(String commandArgs) {
+        String[] split = commandArgs.split(DELIMITER_EVENT, 2);
+        return split;
+    }
+
+    private static void showAddTaskMessage() {
+        System.out.println(DIVIDER_TOP);
+        System.out.println(MESSAGE_PREFIX + MESSAGE_ADD_TASK);
+    }
+
+    private static void echoTask(Task task) {
+        System.out.println(TASK_PREFIX + task.toString());
+    }
+
+    private static void showTaskCountMessage() {
+        System.out.println(MESSAGE_PREFIX + MESSAGE_TASK_COUNT_A + tasks.size() + MESSAGE_TASK_COUNT_B);
+        System.out.println(DIVIDER_BOTTOM);
+    }
+
+    private static void markTaskAsDone(String commandArgs) throws NullCommandArgsException {
+        if (commandArgs == null) {
+            throw new NullCommandArgsException();
+        }
+        else {
+            int index = getDoneIndex(commandArgs);
+            Task task = tasks.get(index);
+            task.markAsDone();
+            showDoneMessage(task);
+        }
+    }
+
+    private static int getDoneIndex(String commandArgs) {
+        return Integer.parseInt(commandArgs) - 1;
+    }
+
+    private static void showDoneMessage(Task task) {
+        System.out.println(DIVIDER_TOP);
+        System.out.println(MESSAGE_PREFIX + MESSAGE_DONE);
+        System.out.println(TASK_PREFIX + task.toString());
+        System.out.println(DIVIDER_BOTTOM);
+    }
+
+    private static void showList() {
+        System.out.println(DIVIDER_TOP);
+        if (tasks.size() == 0) {
+            System.out.println(MESSAGE_PREFIX + MESSAGE_LIST_NO_TASKS);
+        }
+        else {
+            System.out.println(MESSAGE_PREFIX + MESSAGE_LIST);
+            for (Task task : tasks) {
+                System.out.println(MESSAGE_PREFIX + (tasks.indexOf(task)+1) + PERIOD + task.toString());
+            }
+        }
+        System.out.println(DIVIDER_BOTTOM);
+    }
+
+    private static void deleteTask(String commandArgs) throws NullCommandArgsException {
+        if (commandArgs == null) {
+            throw new NullCommandArgsException();
+        }
+        else {
+            int index = getDoneIndex(commandArgs);
+            showDeleteTaskMessage(index);
+            tasks.remove(index);
+            showTaskCountMessage();
+        }
+    }
+
+    private static void showDeleteTaskMessage(int index) {
+        System.out.println(DIVIDER_TOP);
+        System.out.println(MESSAGE_PREFIX + MESSAGE_DELETE_TASK);
+        System.out.println(TASK_PREFIX + tasks.get(index).toString());
+    }
+
+    private static void saveTasks() {
+        String file = FILE_PATH;
+        try {
+            writeToFile(file);
+        } catch (IOException e) {
+            System.out.println(MESSAGE_SOMETHING_WENT_WRONG + e.getMessage());
+        }
+    }
+
+    private static void writeToFile(String filePath) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
```

Comment:
Consider using more descriptive variable names.
Don't be afraid of using longer names as IDE can autocomplete it for you
(At the same time, don't go overboard and use **very** long and cumbersome names)